molo/profiles/models.py

```diff
@@ -85,6 +85,21 @@ class UserProfilesSettings(BaseSetting):
         related_name='+',
         help_text=_('Choose a footer page')
     )
+    activate = models.BooleanField(
```

Comment:
@Ntu2koTrevor you need activate field for each of the field that you added e.g `activate_gender`, `activate_location`...
same applies to the rest